alfresco-health-processor-platform/src/main/amp/config/alfresco/subsystems/HealthProcessor/default/healthprocessor.properties

```
@@ -26,6 +26,8 @@ eu.xenit.alfresco.healthprocessor.plugin.solr-index.endpoints.default.type=Alway
 eu.xenit.alfresco.healthprocessor.plugin.solr-index.endpoints.default.base-uri=http://${solr.host}:${solr.port}/solr/alfresco
 eu.xenit.alfresco.healthprocessor.plugin.solr-index.endpoints.default.indexed-store=workspace://SpacesStore
 
+eu.xenit.alfresco.healthprocessor.plugins.solr.check-transaction=false
```

Please keep the property name consistent with the other properties.

And don't whack it right in the middle of the 2 sets of endpoint configuration for the `solr-index` plugin. You can put it directly after `eu.xenit.alfresco.healthprocessor.plugin.solr-index.enabled`

```suggestion
eu.xenit.alfresco.healthprocessor.plugins.solr-index.check-transaction=false
```

---

This is a configuration property for the `SolrRequestExecutor` and not the solr index validation plugin.

This way other solr plugins can also use this feature of the `SolrRequestExecutor` .  This is why I added it on a different location.

If you insist, I can make your proposed change

---

Well, since the `SolrRequestExecutor` is only used by the `solr-index` plugin, I think it would be correct to put the configuration under that part. Right now, it looks like configuration for a non-existent `solr` plugin.

When other plugins using the same `SolrRequestExecutor` class appear in the future, we can always see what we will do then, maybe even having 2 instances with different configuration is a possibility.